src/3.x/extending.md

```diff
@@ -119,4 +120,55 @@ The autocomplete can contain as many Elasticsearch indexes as you wish. You can
 ],
 ```
 
-You can use `categories.blade.php` as an example for how to properly display new indexes in the autocomplete.
\ No newline at end of file
+### Configuration
+
+There are two ways to define a new additional index. The shorthand way is by only giving the index name and searchable fields, like so:
+
+```php
+'categories' => ['name^3', 'description'],
+```
+
+Alternatively, you can have more control by expanding the configuration like below:
+
+```php
+'blogs' => [
+    'fields' => ['title', 'tags'],
+    'size' => 3,
+    'stores' => ['my_second_store'],
+    'sort' => ['date' => 'desc'],
+],
+```
```

Comment:
This is exactly what's mentioned earlier. I don't think we need 2 examples?

Reply:
I agree, but I figured it might be useful for the clarity having the data as close as possible to the explanation. I could also remove it from the example above, but then it's an incomplete example 🤔

Reply:
Alright, let's stick with this.